### PR TITLE
Fixing description of key words that was over-corrected

### PR DESCRIPTION
**Pull request by moxiegirl**
Adding the phrase "(those 12 or 24 words you recorded)" is confusing.  The count is dependent on the code base, originally 24 was the base and then later 12.  The number of words in the code recorded is immaterial to the user as it is now to the system because the underlying bug was fixed.

A better approach is to supply an example and make no mention of the count *as either count is now accepted*. 

Xref:

#1448 
 https://github.com/blockstack/blockstack-ios/issues/23

Signed-off-by: moxiegirl <mary@blockstack.com>

### CODE REVIEW

File: app/js/sign-in/views/_initial.js [hunk 1]
```diff
@@ -49,8 +49,8 @@ const InitialSignInScreen = ({ next, ...rest }) => {
       children: (
         <React.Fragment>
           <Type.p>
-            Enter your Magic Recovery Code (we sent it to you when you created your ID)
-            or Secret Recovery Key (those 12 or 24 words you recorded).
+            Enter your Magic Recovery Code. This code was sent to you when you created your ID. Alternatively,
+            you can supply your Secret Recovery Key. This key is a sequence of words you recorded, for example, "rabbit, pink, ..."
```
Comment (wbobeirne) [inline app/js/sign-in/views/_initial.js]: The example you give is comma separated, but currently that wouldn't work for the input. I definitely want to make it more robust and allow for those inputs, but we shouldn't change this as-is.

Comment (moxiegirl) [reply app/js/sign-in/views/_initial.js]: I can change the example...I added them by reflex...not intentional. @wbobeirne

Comment (moxiegirl) [reply app/js/sign-in/views/_initial.js]: @wbobeirne done.

**Comment by yknl** [comment]
Thanks for the PR @moxiegirl!

**Comment by moxiegirl** [comment]
Woot! Thanks @yknl & @wbobeirne

**Pull request merged**